docs/_config.yml

```diff
@@ -11,10 +11,41 @@ color_scheme: dark
 exclude:
   - architecture-decisions
 
+baseurl: ""
+# Enable or disable the site search
+# Supports true (default) or false
+search_enabled: true
+search:
+  # Split pages into sections that can be searched individually
+  # Supports 1 - 6, default: 2
+  heading_level: 2
+  # Maximum amount of previews per search result
+  # Default: 3
+  previews: 3
+  # Maximum amount of words to display before a matched word in the preview
+  # Default: 5
+  preview_words_before: 5
+  # Maximum amount of words to display after a matched word in the preview
+  # Default: 10
+  preview_words_after: 10
+  # Set the search token separator
+  # Default: /[\s\-/]+/
+  # Example: enable support for hyphenated search words
+  tokenizer_separator: /[\s/]+/
```

Comment:
This looks like the only value we're setting in this section that isn't using the default value? Is that correct?

I know we've chosen various paths around this in the past (whether to specify default values explicitly or let them be assigned a value by the default setup) - I don't have a problem with any particular value being set, nor the choice to set them specifically. I just want to put this change in context.

Reply:
Yeah in this case I felt it was useful to understand what the configuration options were even if we were leaving defaults. I'm also not entirely sure why this particular setting isn't using the default. I suspect the default changed between our initial setup and now upstream.